update-gyp.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+import tarfile
+import tempfile
+import urllib.request
+
+BASE_URL = "https://github.com/nodejs/gyp-next/archive/"
+CHECKOUT_PATH = os.path.dirname(os.path.realpath(__file__))
+CHECKOUT_GYP_PATH = os.path.join(CHECKOUT_PATH, 'gyp')
+
+parser = argparse.ArgumentParser()
+parser.add_argument("tag", help="gyp tag to update to")
+args = parser.parse_args()
+
+tar_url = BASE_URL + args.tag + ".tar.gz"
+
+changed_files = subprocess.check_output(["git", "diff", "--name-only"]).strip()
+if len(changed_files) is not 0:
```

Comment:
```suggestion
if changed_files:
```
Identity is not the same thing as equality in Python so __use ==/!= to compare constant literals (str, bytes, int, float, tuple)__ to avoid [SyntaxWarnings on Python >= 3.8](https://docs.python.org/3/whatsnew/3.8.html#porting-to-python-3-8).

% `python3.8`
```
>>> 0 is 0
<stdin>:1: SyntaxWarning: "is" with a literal. Did you mean "=="?
```